packages/forms/src/UIForm/UIForm.component.test.js

```diff
@@ -332,18 +350,39 @@ describe('UIForm component', () => {
 
 		it('should call submit callback when form is valid', () => {
 			// given
-			const validProperties = {
-				...data.properties,
-				lastname: 'This has at least 10 characters',
-				firstname: 'This is required',
-			};
 			const wrapper = mount(<UIFormComponent {...data} {...props} properties={validProperties} />);
 
 			// when
 			wrapper.instance().onSubmit(submitEvent);
 
 			// then
 			expect(props.onSubmit).toBeCalled();
+			expect(props.setErrors).toHaveBeenCalledWith(submitEvent, {}, noop);
+		});
+	});
+
+	describe('#focusFirstError', () => {
+		it('should focus the first element', () => {
+			const focus = jest.fn();
+			const querySelector = jest.fn(() => ({
+				focus,
+			}));
+			const wrapper = shallow(<UIFormComponent {...data} {...props} />);
+			wrapper.instance().formRef = { querySelector };
+			wrapper.instance().focusFirstError();
+
+			expect(querySelector).toHaveBeenCalledWith(INVALID_FIELD_SELECTOR);
+			expect(focus).toHaveBeenCalled();
+		});
+	});
+
+	describe('#setFormRef', () => {
```

Comment:
Remove this test, it's about internal implem, it's indirectly tested with the focus

Reply:
done